playbooks/aws/openshift-minion/launch.yml

```diff
@@ -65,5 +70,11 @@
       delay: 10
       with_items: ec2.instances
 
-# Apply the configs, seprate so that just the configs can be run by themselves
-- include: config.yml
+- name: "Configure instances"
```

Comment:
Then cluster.sh should be changed. We use cloud.rb independent of cluster.sh, and these are playbooks that can be run directly with ansible (in other words completely independent of both cloud.rb and cloud.sh). We intend to eventually run these out of Ansible tower, which is yet another playbook launch mechanism. All of these are outside of cluster.sh.

The intention here is that launch.yml be just enough to actually launch the instances, and run a config using the config.yml playbook. It is not intuitive that a launch wouldn't also include an initial config.